### PR TITLE
25 create login page

### PR DESCRIPTION
**Pull request by Ozrei**

### CODE REVIEW

File: keyform/templates/keyform/base.html [hunk 1]
```diff
@@ -24,10 +24,13 @@
                 <div class="title-bar">
                     <img src="{% static 'keyform/logo.jpg' %}"/>
                     <div class="title-bar-title">{% trans 'Keyform' %}</div>
+                    {% block links %}
```
Comment (shaneallgeier) [inline keyform/templates/keyform/base.html]: Improperly indented. Also, change it from "links" to "navigation." And remember to do that anywhere that block is being used.

File: keyform/views.py [hunk 2]
```diff
@@ -7,22 +7,29 @@
...
 from django.urls import reverse_lazy
 from django.http import HttpResponseRedirect
+from django.contrib.auth.mixins import LoginRequiredMixin
 
-class HomeView(ListView):
+class HomeView(LoginRequiredMixin, ListView):
     model = Request
     template_name = "keyform/home.html"
+    login_url = "login"
+    redirect_field_name = "redirect_to"
```
Comment (shaneallgeier) [inline keyform/views.py]: `login_url` and `redirect_field_name` aren't necessary here (or in any of the following view classes)

File: keyform/static/keyform/css/login.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
+div.large-4 {
```
Comment (shaneallgeier) [inline keyform/static/keyform/css/login.css]: You probably shouldn't use this as a selector. Try setting an extra, custom class on line 15 in login.html. Maybe `login-container`?

File: keyform/static/keyform/css/login.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.button {
+    background-color: #80151b;
+}
+
+li {
+    padding-top: 10px;
+    color: red;
+}
```
Comment (enki211) [inline keyform/static/keyform/css/login.css]: Newline

**Comment by enki211** [comment]
Looks good

**Pull request merged**